editor/plugin_config_dialog.cpp

```diff
@@ -78,17 +111,30 @@ void PluginConfigDialog::_on_confirmed() {
 		if (lang_name == GDScriptLanguage::get_singleton()->get_name()) {
 			// Hard-coded GDScript template to keep usability until we use script templates.
 			Ref<Script> gdscript = memnew(GDScript);
-			gdscript->set_source_code(
+			String source =
 					"tool\n"
 					"extends EditorPlugin\n"
-					"\n"
+					"\n";
+			if (model >= 2) {
+				source +=
+						TTR("# This plugin uses the new universal model. Things you need to know:\n") +
+						TTR("# - You can place it inside a specific project, at <project_dir>/addons/ or,\n") +
+						TTR("#   to make it user-wide (available for any project you open under your OS user),\n") +
+						TTR("#   put it instead at <user_editor_settings>/addons/.\n") +
+						TTR("#   In either case it will be available in the Godot file system as addons://plugin_name.\n") +
+						TTR("# - To reference resources, you must use that kind of paths. For instance:\n") +
+						"#   var scene = load(\"addons://my_plugin/scene.tscn\")\n"
+						"\n";
```

Comment:
The comment should respect the 80 characters limit. Also, all other code comments created by the editor aren't translated, so with this the other should probably follow suit then.